libenkf/src/gen_data_config.c

```diff
@@ -397,6 +397,49 @@ void gen_data_config_assert_size(gen_data_config_type * config , int data_size,
   pthread_mutex_unlock( &config->update_lock );
 }
 
+
+static void update_config_to_datamask(
+  gen_data_config_type * config,
+  const forward_load_context_type * load_context,
+  const bool_vector_type * data_mask,
+  int report_step)
+{
+  // Is this the first ensemble member loading for this particular report_step?
+  if (config->active_report_step != report_step) {
+    bool_vector_reset(config->active_mask);
+    bool_vector_iset(config->active_mask,
+                     int_vector_iget(config->data_size_vector, report_step) - 1,
+                     true);
+    config->mask_modified = true;
+  }
+
+  // set config inactive according to data_mask
+  for (int i = 0; i < bool_vector_size(data_mask); ++i) {
+    if (bool_vector_iget(data_mask, i))
+      continue;
+    bool_vector_iset(config->active_mask, i, false);
+    config->mask_modified = true;
+  }
+
+  if (!config->mask_modified)
+    return; // nothing to do
+
+  // The global mask has been modified after the last load;
+  // i.e. we update the on-disk representation.
+  char * filename = util_alloc_sprintf("%s_active", config->key);
+  FILE * stream = enkf_fs_open_case_tstep_file(
```

Comment:
The mother of all irrelevant comments; I would like if this was formatted with:
1. The first argument, `forward_load_context_get_...` one line up.
2. Everything indented to the column of the opening `(`
That way I think it is easier to see that this is indeed one (too) long function call, and not regular lines of code.